core_api/src/routes/chat.py

```diff
@@ -195,12 +199,12 @@ async def rag_chat_streamed(
     chat_request = ChatRequest.model_validate_json(request)
     logger.debug("chat request from django-app: %s", chat_request)
 
-    chain, params = await build_chain(chat_request, user_uuid, llm, vector_store)
+    chain, params, route = await build_chain(chat_request, user_uuid, llm, vector_store)
 
     async for event in chain.astream_events(params, version="v1"):
         kind = event["event"]
         if kind == "on_chat_model_stream":
-            await websocket.send_json({"resource_type": "text", "data": event["data"]["chunk"].content})
+            await websocket.send_json({"resource_type": "text", "route": route, "data": event["data"]["chunk"].content})
```

Comment:
Will this sent the route as part of every fragment? Might it be better to send it once?